test/controllers/slam_controller_test.exs

```diff
@@ -1,6 +1,30 @@
 defmodule Healthlocker.SlamControllerTest do
   use Healthlocker.ConnCase
-  alias Healthlocker.User
+  alias Healthlocker.{User, EPJSUser, ReadOnlyRepo}
+
+  @valid_attrs %{
+    first_name: "Kat",
```

Comment:
so the person can have 4 (_four_) names?
+ `first_name`
+ `last_name`
+ `forename`
+ `surname`

or is this just while we are _migrating_ the data?